backend/core/views.py

```
@@ -444,20 +444,38 @@
         return Response(my_map)
 
 
+class AssetFilter(df.FilterSet):
+    exclude_childrens = df.ModelChoiceFilter(
+        queryset=Asset.objects.all(),
```

Ruff (0.8.2)

`Asset` may be undefined, or defined from star imports

(F405)